src/pcapi/core/users/factories.py

```diff
@@ -63,6 +63,35 @@ def deposit(obj, create, extracted, **kwargs):  # pylint: disable=no-self-argume
         return DepositFactory(user=obj, **kwargs)
 
 
+class ProFactory(BaseFactory):
+    class Meta:
+        model = pcapi.core.users.models.User
+
+    email = factory.Sequence("ma.librairie{0}@example.com".format)
+    address = factory.Sequence("{0} rue des cinémas".format)
```

Comment:
```suggestion
    email = factory.Sequence("ma.librairie{}@example.com".format)
    address = factory.Sequence("{} rue des cinémas".format)
```
Sinon on ne peut pas en créer plus de 10, c'est dommage :)